bucket/openrgb.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.6",
+    "description": "Open source RGB lighting control that doesn't depend on manufacturer software.",
+    "homepage": "https://openrgb.org",
+    "license": "GPL-2.0-or-later",
```

Comment:
Source? Where is clearly stated it is `or later`?

Reply:
Didn't know what to actually put here. But I guess it is not `or later` unless specified.